app/controllers/admin/loans_controller.rb

```diff
@@ -63,6 +73,12 @@ def destroy
     end
   end
 
+  def prepare_event(cal_event)
+    content = render_to_string(partial: "admin/calendar/event", locals: {cal_event: cal_event}).html_safe
+    cal_event[:title] = content
+    @events.push(cal_event)
```

Comment:
This code is copied and pasted verbatim from the previous controller. This is never ok. Let's talk about the correct way to fix this.